CivicActions/civicactions-homesite
# MAHOME-1139: Remove Department of Education Logo

## What's Changed
- Removed the dept. of ed logo from the homepage.
- Updated README to reflect updates from Gatsby upgrade.

## Testing Locally
1. Run `yarn install`.
2. Confirm that .env file exists with correct credentials, then run `source .env`.
3. Run `npm run start`.

![Screenshot 2023-12-13 at 2 56 51 PM](https://github.com/CivicActions/civicactions-homesite/assets/12264276/a8994dd8-ea64-4e14-ac76-65ef04f47a2e)
